toot.gemspec

```diff
@@ -1,29 +1,37 @@
-# coding: utf-8
-lib = File.expand_path('../lib', __FILE__)
+# frozen_string_literal: true
+
+lib = File.expand_path('lib', __dir__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'toot/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = "toot"
+  spec.name          = 'toot'
   spec.version       = Toot::VERSION
-  spec.authors       = ["Travis Petticrew"]
-  spec.email         = ["travis@petticrew.net"]
+  spec.authors       = ['Travis Petticrew']
```

Comment:
Let's go ahead and change this to `Watermark Dev` like our other gems (including email)